backend/tests/homepageController.test.ts

```diff
@@ -552,6 +553,96 @@ describe('Homepage', () => {
 
       expect(res.redirect).toHaveBeenCalledWith('/videolink')
     })
+
+    describe('Manage incentives tile', () => {
+      beforeEach(() => {
+        config.apis.incentives.ui_url = 'http://incentives'
+      })
+
+      it('should not render home page with the Incentives tile if user has excluded caseload selected', async () => {
+        config.apis.incentives.excludedCaseloads = 'CADM_I'
+        req = {
+          session: {
+            userDetails: {
+              staffId: 1,
+              activeCaseLoadId: 'CADM_I',
+            },
+          },
+        }
+
+        await controller(req, res)
+
+        expect(res.render).toHaveBeenCalledWith(
+          'homepage/homepage.njk',
+          expect.objectContaining({
+            tasks: [],
+          })
+        )
+
+        // set this back to empty list
+        config.apis.incentives.excludedCaseloads = ''
+      })
+
+      it('should not render home page with the Incentives tile if user is an LSA', async () => {
```

Comment:
You don't need to do this. You wouldn't want to exclude users because of these roles.  These roles can be assigned to any user and then they wouldn't get the tile.  The caseload is good enough and this size of the location dropdown above